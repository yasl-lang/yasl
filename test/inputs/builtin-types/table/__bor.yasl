
let x = { .a: 10, .b: 11 }
let y = { .b: 100, .c: 12}

echo x | y
