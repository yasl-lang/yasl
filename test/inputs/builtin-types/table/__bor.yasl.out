{a: 10, b: 100, c: 12}
